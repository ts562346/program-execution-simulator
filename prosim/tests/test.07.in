test1 4
LOOP 4
  DOOP 2
END
HALT

