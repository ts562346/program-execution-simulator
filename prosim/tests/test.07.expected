00000: DOOP
00002: DOOP
00004: DOOP
00006: DOOP
00008: HALT

DOOP count : 4
DOOP time  : 8
BLOCK count: 0
BLOCK time : 0
